src/Modules/popupMenu.js

```diff
@@ -0,0 +1,54 @@
+import { getComments, postAComment } from './comments.js';
+import { arr } from './tvApi.js';
+
+const removeMenu = (menu) => {
+  document.body.removeChild(menu);
+  document.body.style.overflow = 'scroll';
+};
+
+const displayMenu = (btnId) => {
+  const id = btnId.replace(/\D+/, '') - 1;
+  const popupMenu = document.createElement('section');
+  popupMenu.classList.add('popupMenuBackground');
+  popupMenu.innerHTML = `
+  <div class="popupMenu">
+  <button class="exitBtn" id="popupExitBtn">X</button>
+  <div>
+    <img src="${arr[id].image.medium}" class="popupMenuImg">
+  </div>
+  <div class="movie">
+    <h3 class="movieTitle">${arr[id].name}</h3>
+    <p class="movieInfo pMargin">genres : ${arr[id].genres}</p>
+    <div class="infoDiv">
+      <p class="movieInfo pMargin">language : ${arr[id].language}</p>
+      <p class="movieInfo pMargin">status : ${arr[id].status}</p>
+    </div>
+    <div class="infoDiv">
+      <p class="movieInfo pMargin">network : ${arr[id].network.name}</p>
+      <p class="movieInfo pMargin">rating : ${arr[id].rating.average}</p>
+    </div>
+  </div>
+  <div class="commentsTitleDiv" id="commentsTitleDiv">
+    <h3 id="commentsTitle">Comments : 0</h3>
+  </div>
+  <div class="comments" id="commentsDiv">  
+    <p id="firstComment"></p>
+  </div>
+  <form>
+    <title>Add A Comment</title>
+    <input id="fName" class="fName" placeholder="Your name">
+    <textarea id="fComment" class="fComment" placeholder="Your insigths"></textarea>
+    <button id="submitComment" class="submitBtn" type="button">Comment</button>
+  </form>
+</div>
```

Comment:
- [ ] Hi, The project requires that you should adhere to the given in terms of data displayed on the Page.  When I fill out the comment form with my `name` and `comment`, the app displays only the comment and hides the name. I would advise you to work out and fix the mistake. 
- [ ] Can you try to display the `date` because it is also part of the `data being displayed on the original page`.
I posted the pictures below for you to understand what I am talking about.
- **I fill out the Form:**
![Capture](https://github.com/IsmaelMastronardi/Capstone2/assets/63972670/239ac6ab-9d78-4c41-8d57-e773f3e8340c)

- **I submit the form**(NAME disappear)
![Capture1](https://github.com/IsmaelMastronardi/Capstone2/assets/63972670/24a51bef-de96-4499-aced-6dae5a53ebf2)
- **Original design**. 
![Capture](https://github.com/IsmaelMastronardi/Capstone2/assets/63972670/a65ef2d2-33f9-46da-8589-029b207cd89c)